.github/workflows/docker-build-and-push.yaml

```
@@ -0,0 +1,56 @@
+name: docker-build-and-push
+
+on:
+  push:
+    tags:
+      - v*
```

How about making the pattern follow semver?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

```suggestion
      - v[0-9]+.[0-9]+.[0-9]+
```

---

Thank you, but actually, we have to consider the cases with suffixes such as `v0.1.2-beta`.

---

Since I think we don't create tags so often, I feel this is enough for now.
If we want to use some tags like `v-foo-bar`, we can change the condition later.

---

OK. I understood.